suite-native/config/src/supportedNetworks.ts

```diff
@@ -1,43 +1,52 @@
 import { A } from '@mobily/ts-belt';
 
+import { isTestnet } from '@suite-common/wallet-utils';
 import { Network, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
 
-const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol'];
-
-export const supportedPortfolioTrackerNetworks = {
-    mainnet: ['btc', 'eth', 'ltc', 'doge', 'etc', 'ada', 'bch', 'xrp', 'zec'] as NetworkSymbol[],
+export const networkSymbolsWhitelistMap = {
+    mainnet: [
+        'btc',
```

Comment:
I think this will allow import coins into portfolio that were not supposed to be there (btg, dgb, ...) for portfolio device, but should be there for discovery.

Reply:
Yes, I was just fixing this. Only realised it now. https://github.com/trezor/trezor-suite/pull/10482/commits/f81ebd4e66e1a03359da1201170b8fe6740086db